### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by olferuk**
Added info about newly added operators

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,13 +107,15 @@ spec.source_files = 'Argo/**/*.{h,swift}', 'Carthage/Checkouts/Runes/Source/Rune
...
 
-Importing Runes introduces 3 new operators and one global function:
+Importing Runes introduces several new operators and one global function:
 
 - `<^>` (pronounced "map")
 - `<*>` (pronounced "apply")
 - `>>-` (pronounced "flatMap") (left associative)
 - `-<<` (pronounced "flatMap") (right associative)
 - `pure` (pronounced "pure")
+- `>->` (pronounced "Kleisli arrow") (left associative)
```
Comment (gfontenot) [inline README.md]: This is technically correct, but maybe we could call it "Monadic compose" instead? Seems a _little_ more approachable in my opinion.

Comment (olferuk) [inline README.md]: Ok, agreed

**Comment by gfontenot** [comment]
Thanks! I've squashed and merged this as d7c2bf1

**Comment by gfontenot** [comment]
lol and screwed up the commit

_really_ merged as 9f5c8ca

**Pull request closed without merging**